homeassistant/components/template/trigger_entity.py

```diff
@@ -50,7 +50,9 @@ def referenced_blueprint(self) -> str | None:
     @callback
     def _render_script_variables(self) -> dict:
         """Render configured variables."""
-        return self.coordinator.data["run_variables"]
+        if self.coordinator.data is None:
+            return {}
```

Comment:
Should this case be covered by a dedicated test in `test_trigger_entity.py`?

Reply:
Yes, probably.  Add that to this or separate PR?